src/MSBuild/Resources/Strings.resx

```diff
@@ -687,6 +687,23 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_32_RestorePropertySwitch" UESanitized="false" Visibility="Public">
```

Comment:
There's a `HelpMessage_32_` way later in the file (not sure why not right here . . .). Make this 33?

Reply:
Good catch!